examples/go/docker/cdktf.json

```diff
@@ -0,0 +1,8 @@
+{
+    "language": "go",
+    "app": "go run main.go",
+    "terraformProviders": [
+        "kreuzwerker/docker@~> 2.11"
+    ],
```

Comment:
It would be nice to have a `package.json` like in the [other examples](https://github.com/hashicorp/terraform-cdk/blob/fa335ed8f2f68bc233d70248336fc90f2b6560a2/examples/python/azure/package.json) to allow building this via lerna.

Reply:
Added one for the docker example and added a todo to add one to the AWS example as soon as the out of memory error is addressed.